React/wbooks/src/services/authService.js

```diff
@@ -17,4 +17,22 @@ const retrieveUserData = async (name, password) => {
 const retrieveUserFromSession = () => {
   return localStorage.retrieveUserFromLocalStorage();
 }
-export default {retrieveUserData, retrieveUserFromSession};
+
+const registerUser = async(name, password, confirmPassword, firstName, lastName) => {
+  await api.post('/users', {
+    user: {
+      email: name,
+      password: password,
+      confirm_password: confirmPassword,
+      first_name: firstName,
+      last_name: lastName,
+      locale: 'en'
+    }
+  }).then(response => {
+    return true;
+  }).catch(error => {
+    throw error;
+  });
+}
+
+export default {retrieveUserData, retrieveUserFromSession, registerUser};
```

Comment:
You can also export each const, por example `export const registerUser =  ...` and you avoid the export default.